app/lib/domicile_resolver.rb

```diff
@@ -6,7 +6,7 @@ def self.hesa_code_for_country(iso_country_code)
     when 'CY' then 'XC'
     when 'XK' then 'QO'
     else
-      iso_country_code
+      iso_country_code[0..1]
```

Comment:
Should we only limit the hesa code in the API as opposed to everywhere else in the app?

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/6946/commits/0c86c2fb1a498448bcbe6f15f35bdca1b08de4b1 restricts this to Vendor API responses